make/common/NativeCompilation.gmk

```diff
@@ -53,9 +59,9 @@ define GetSymbols
         $(CUT) -d ' ' -f 3 >>  $$(@D)/$$(basename $$(@F)).symbols || true;\
   else \
     $(ECHO) "Getting symbols from nm"; \
-    $(NM) -m $$($1_TARGET) | $(GREP)  "__TEXT" | \
+    $(MNM) $$($1_TARGET) | $(GREP)  "__TEXT" | \
         $(EGREP) -v "non-external|private extern|__TEXT,__eh_frame" | \
-        $(SED) -e  's/.* //' > $$(@D)/$$(basename $$(@F)).symbols; \
+        $(SED) -e  's/.* //' > $$(@D)/$$(basename $$(@F)).symbols || true; \
```

Comment:
Can you explain why you need to add "|| true" ?

Reply:
If this is not added, the build fails with
`/bin/bash: -c: line 1: syntax error: unexpected end of file`

I think this is in case there are no symbols, and this command is part of a flow which expects all commands to return true (which does not happen when the command is not producing any output)

I noticed the` || true` being included in other sections of the same file.